examples/body.ts

```diff
@@ -3,10 +3,10 @@ import { createApp, createRouter, readJSONBody } from "h3";
 export const app = createApp();
 
 const router = createRouter()
-  .get("/", () => "use POST method to try!")
+  .get("/", () => "use GET method to try!")
```

Comment:
This, actually, is not a typo. It encourage the user to try and send POST request